pydicom/pixel_data_handlers/rle_handler.py

```diff
@@ -60,6 +70,9 @@ def get_pixeldata(dicom_dataset):
     if 'PixelData' not in dicom_dataset:
         raise TypeError("No pixel data found in this dataset.")
 
+    if frame_list is not None:
+        msg = ("Reading individual frames with GDCM is not yet supported.")
+
```

Comment:
Refers to gdcm here but seems to still be in rle_handler - just a typos?

Reply:
Just a typo. Thanks for spotting it!